streams/src/test/java/org/apache/kafka/streams/state/internals/CachingWindowStoreTest.java

```diff
@@ -104,6 +104,7 @@ public void closeStore() {
         cachingStore.close();
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
In this case, it seems like we should just migrate away from the deprecated method. This test isn't specifically verifying the behavior of the deprecated method itself, it just happens to need to put into a window store as part of the test.

What do you think?

Reply:
removed